paddle/framework/tensor_impl.h

```diff
@@ -148,5 +148,13 @@ inline Tensor& Tensor::Resize(const DDim& dims) {
 
 inline const DDim& Tensor::dims() const { return dims_; }
 
+template <typename T>
+inline Tensor ReshapeToMatrix(const Tensor& src, int num_col_dims) {
```

Comment:
`inline` is not needed in `class method`. It will be compiler's choice whether inline or not.

Reply:
It's not a class method. It's a global function.